src/vma/dev/net_device_table_mgr.cpp

```diff
@@ -299,12 +299,14 @@ int net_device_table_mgr::map_net_devices()
 	return 0;
 }
 
-bool net_device_table_mgr::verify_bond_ipoib_or_eth_qp_creation(struct ifaddrs * ifa) {
+bool net_device_table_mgr::verify_bond_ipoib_or_eth_qp_creation(struct ifaddrs * ifa)
+{
+	char base_ifname[IFNAMSIZ];
+	get_base_interface_name((const char*)(ifa->ifa_name), base_ifname, sizeof(base_ifname));
```

Comment:
Assuming no need to check return value of get_base_interface_name() since the next call to get_bond_slaves_name_list() would fail if needed